index.html

```diff
@@ -14,41 +14,23 @@ <h1 id="title">freeCodeCamp Survey Form</h1>
   
     <form id="survey-form">
       <label id="name-label">Name</label><br />
-      <input
```

Comment:
I think it would be better if you wrote your input element in one single line.

Reply:
- [ ] Good job so far but please make sure to always tell the reason, **WHY** you suggest something or request a change :smile:  For example: "Please adjust margin to 5px for label element because that would improve the page more accessibility" :+1:

Reply:
- [ ] Hey @Salwa99 👋 You have done well in implementing most of the changes suggested by the previous reviewer however, you did not state WHY it would be better to write the input element in a single line and why having blank spaces in your code isn't a good idea. 
For Example, "I think it would be better if you wrote your input element in one single line thus removing blank spaces in order to minimize noise in commits". 👍